GitCommands/Settings/AppSettings.cs

```diff
@@ -41,6 +41,15 @@ public enum ShorteningRecentRepoPathStrategy
         MiddleDots
     }
 
+    public enum IgnoreWhitespaceMethod
```

Comment:
Accepted and changed PR accordingly.
With the only exception of the suggested xml-doc and URL for `IgnoreWhitespaceKind.None`, because those suggested came from [--color-moved-ws=no](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---color-moved-wsltmodesgt), which is similar.
`IgnoreWhitespaceKind.None` is the default behavior of `git diff` when no additional flag is provided. Thus there's no explicit reference on git-scm.com.